pkg/security/proto/security_profile/v1/activity_dump.proto

```diff
@@ -33,6 +33,15 @@ message Metadata {
     string serialization = 14;
 }
 
+message SecurityProfile {
+    string status = 1;
+    string version = 2;
+    Metadata metadata = 3;
```

Comment:
Should we split/reduce the used Metadata to avoid having useless fields sent back as profile ? (agent version/commit, kernel, linux, arch, start, end, size ..)